ktorm-support-postgresql/src/main/kotlin/org/ktorm/support/postgresql/BulkInsert.kt

```diff
@@ -223,3 +224,217 @@ public class BulkInsertOrUpdateOnConflictClauseBuilder : PostgreSqlAssignmentsBu
         )
     }
 }
+
+/**
+ * Bulk insert expression, represents a bulk insert statement in PostgreSQL.
+ *
+ * For example:
+ *
+ * ```sql
+ * insert into table (column1, column2) values (?, ?), (?, ?), (?, ?)...
+ * `on conflict (key) do update set (...) returning ...
+ * ```
+ *
+ * @property table the table to be inserted.
+ * @property assignments column assignments of the bulk insert statement.
+ * @property conflictColumns the index columns on which the conflict may happens.
+ * @property updateAssignments the updated column assignments while key conflict exists.
+ * @property returningColumns the columns to returning.
+ */
+public data class BulkInsertReturningExpression(
```

Comment:
If I recall correctly, the reason why I did a new method is because its `return` type is different, and it would make a breakable change. Also, returning has a performance overhead that you might not want to have as default.

Reply:
Oh, we can create a new method, we have to do that in this way because backward compatibility is important. What I mean is they can share the same expression class, we just need to add `returningColumns` to `BulkInsertExpression`, with a default value `emptyList()`.

Reply:
Ok, I'll try to make this change when I have time!